src/components/aktivitetskrav/vurdering/AktivitetskravVurderingAlert.tsx

```diff
@@ -11,6 +11,17 @@ import React, { ReactElement } from "react";
 import { avventVurderingArsakTexts } from "@/data/aktivitetskrav/aktivitetskravTexts";
 import { AktivitetskravAlertstripe } from "@/components/aktivitetskrav/AktivitetskravAlertstripe";
 import { BodyLong, BodyShort, Label } from "@navikt/ds-react";
+import { usePersonoppgaverQuery } from "@/data/personoppgave/personoppgaveQueryHooks";
+import { hasUbehandletPersonoppgave } from "@/utils/personOppgaveUtils";
+import { PersonOppgaveType } from "@/data/personoppgave/types/PersonOppgave";
+
+const texts = {
+  forhandsvarselInfoBody:
+    "Når fristen har passert vil det dukke opp en hendelse til deg i oversikten og du må vurdere aktivitetskravet.",
```

Comment:
Er det litt redundant å ha med at de må vurdere aktivitetskravet? 🤔
```suggestion
    "Når fristen har passert vil det dukke opp en hendelse til deg i oversikten.",
```

Reply:
Jo, kan være enig i det. Kan ta en ekstra runde med Stine på det :+1: